lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -565,7 +567,8 @@ if (require.main === module) {
   const coreStrings = collectAllStringsInDir(path.join(LH_ROOT, 'lighthouse-core'));
   console.log('Collected from LH core!');
 
-  const stackPackStrings = collectAllStringsInDir(path.join(LH_ROOT, 'stack-packs/packs'));
+  const stackPacksRoot = `${LH_ROOT}/node_modules/lighthouse-stack-packs/packs`;
```

Comment:
node modules paths are weird so I prefer using something like 

```suggestion
  const stackPacksRoot = `${require.resolve('lighthouse-stack-packs')}/packs`;
```

Reply:
haha @connorjclark said I had to change it into LH_ROOT :joy:

Reply:
hahah ah i see that now. :)

sorry connor but we've seen node_module paths be unreliable to guess. so best to rely on `require.resolve()`

Reply:
good point, for ex: if you yarn link this pkg